scripts/ts-bin/testdata/foo.csv

```diff
@@ -0,0 +1,4 @@
+foo, bar
```

Comment:
Is the space before 'bar' intentional? I'm actually surprised this work-- is there some magic I'm not aware of that trims leading and trailing whitespace?

Reply:
Indeed, CSV is a magic complex loosely defined format. :) 
But, as it happens, I don't need the testdata directory, so that's removed as the testdata in actually inline.